### ISSUE TITLE
[enhancement]: Do not suggest to use npm

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.js-controller#2955]
### No existing issues.

- [X] There is no existing issue for my request.

### Description

If an adapter is not available at repository th following Test is displayed:

```
$ iobroker add flexcharts auto --host MiniPC --debug

host.MiniPC Adapter "system.adapter.flexcharts.0" is stopped.
host.MiniPC Unknown packetName flexcharts. Please install packages from outside the repository using npm!
ERROR: Process exited with code 12
````

see https://forum.iobroker.net/topic/76137/test-adapter-flexcharts-stapeldiagramme-und-mehr/89

I would suggest to change the text `Please install packages from outside the repository using npm!` to something not recommending npm, i.e. `Please install packages from outside the repository using iobroker url ...!`

### Why?

We should avoid to instruct users to use npm to install anything as this might cause additional problems if used incorrectly or inside the wrong directory.

**Comment by foxriver76** [comment]
https://github.com/ioBroker/ioBroker.js-controller/issues/2664

Isn’t this fixed in current beta 7.0.x?

**Comment by mcm1957** [comment]
Upps - sorry.
I stumbled about the message in the forum and did not remember my own issue. Shame on me.
